examples/compute-dg-api.ipynb

```diff
@@ -0,0 +1,1355 @@
+{
```

Comment:
<p>Can you add an extra bit here about how MLEs are not possible without a fully connected graph?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/cinnabar/pull/160/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>